NEWS

```diff
@@ -26,6 +26,9 @@ PHP                                                                        NEWS
     (David Carlier)
   . Fixed bug GH-15087 (IntlChar::foldCase()'s $option is not optional). (cmb)
 
+- Intl:
```

Comment:
note: This is definitively a weird entry, derived from just above. should not be..